src/main/kotlin/org/rust/ide/newProject/ui/RsNewProjectPanel.kt

```diff
@@ -71,11 +83,27 @@ class RsNewProjectPanel(
     }
 
     private val selectedTemplate: RsProjectTemplate
-        get() = templateList.selectedValue
+        get() {
+            if (templateList.isSelectionEmpty) {
+                templateList.selectedIndex = 0
+            }
+            return templateList.selectedValue
+        }
```

Comment:
Looks like this code is not necessary due to `selectedIndex = 0` on the creation of `templateList` (so there is a valid selection at any time). Am I missing something else?